recipes/gtk4-layer-shell/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -ex
+
+export LD_LIBRARY_PATH=$PREFIX/lib
+meson setup -Dexamples=false -Ddocs=false -Dintrospection=false -Dvapi=false -Dsmoke-tests=false -Dtests=true -Dc_link_args='-ldl' --prefix=$PREFIX build
```

Comment:
Please use `MESON_ARGS` here, this should also fix the `lib64` problem, see https://github.com/search?q=org%3Aconda-forge%20%24%7BMESON_ARGS%7D&type=code .